src/app/cluster/details/cluster/edit-cluster/component.ts

```diff
@@ -197,6 +210,57 @@ export class EditClusterComponent implements OnInit, OnDestroy {
       : AuditPolicyPreset.Custom;
   }
 
+  onLabelsChange(labels: Record<string, string>): void {
+    this.labels = labels;
+  }
+
+  onPodNodeSelectorAdmissionPluginConfigChange(config: Record<string, string>): void {
+    this.podNodeSelectorAdmissionPluginConfig = config;
+
+    this.handleClusterDefaultNodeSelector(config);
+  }
+
+  handleClusterDefaultNodeSelector(nodeConfig: Record<string, string>): void {
```

Comment:
This method is complete duplicate can we move some common place i am not sure but wdyt? some common code related to cluster (wizard/edit)